data/presets/seamark/offshore_platform.json

```
@@ -0,0 +1,40 @@
+{
+    "icon": "maki-harbor",
+    "fields": [
+        "name",
+        "seamark/platform/category",
+        "seamark/radio_station/mmsi"
+    ],
+    "geometry": [
+        "point",
+        "vertex",
+        "area"
+    ],
+    "tags": {
+        "seamark:type": "platform"
+    },
+    "addTags": {
+        "seamark:type": "platform",
+        "man_made": "offshore_platform"
+    },
+    "removeTags": {
+        "seamark:type": "*",
+        "man_made": "offshore_platform"
+    },
+    "terms": [
+        "bore",
+        "drilling rig",
+        "drilling rig",
+        "offshore platform",
+        "oil derrick",
+        "oil drill",
+        "oil horse",
+        "oil pump",
+        "oil rig",
+        "oil rig",
+        "petroleum well",
+        "platform",
+        "pumpjack"
```

Duplicate terms can be removed:

```suggestion
        "bore",
        "drilling rig",
        "offshore platform",
        "oil derrick",
        "oil drill",
        "oil horse",
        "oil pump",
        "oil rig",
        "petroleum well",
        "platform",
        "pumpjack"
```